sensorserviceutils/sensorserviceutils_test.go

```diff
@@ -218,37 +432,506 @@ func TestParse(t *testing.T) {
+					UID:                    10013,
+					SensorNumber:           24,
+					PendingFlush:           0,
+					SamplingPeriodUs:       250000,
+					BatchingPeriodUs:       0,
+					HasSensorserviceRecord: true,
+					Source:                 "Sensorservice Dump",
+				},
+				2: {
+					Number:                 2,
+					OperatingMode:          "NORMAL",
+					PackageName:            "com.google.android.location.collectionlib.w",
+					UID:                    10013,
+					SensorNumber:           4,
+					PendingFlush:           0,
+					SamplingPeriodUs:       66670,
+					BatchingPeriodUs:       0,
+					HasSensorserviceRecord: true,
+					Source:                 "Sensorservice Dump",
+				},
+				3: {
+					Number:                 3,
+					OperatingMode:          "NORMAL",
+					PackageName:            "com.android.server.display.AutomaticBrightnessController",
+					UID:                    1000,
+					SensorNumber:           7,
+					PendingFlush:           0,
+					SamplingPeriodUs:       200000,
+					BatchingPeriodUs:       100000,
+					HasSensorserviceRecord: true,
+					Source:                 "Sensorservice Dump",
+				},
+				4: {
+					Number:                 4,
+					OperatingMode:          "NORMAL",
+					PackageName:            "com.android.server.policy.WindowOrientationListener",
+					UID:                    1000,
+					SensorNumber:           31,
+					PendingFlush:           0,
+					SamplingPeriodUs:       20000,
+					BatchingPeriodUs:       0,
+					HasSensorserviceRecord: false,
+					Source:                 "Sensorservice Dump",
+				},
+				5: {
+					Number:                 5,
+					OperatingMode:          "NORMAL",
+					PackageName:            "civ",
+					UID:                    10182,
+					SensorNumber:           1,
+					PendingFlush:           0,
+					SamplingPeriodUs:       -1,
+					BatchingPeriodUs:       -1,
+					HasSensorserviceRecord: true,
+					Source:                 "Sensorservice Dump",
+				},
+			},
+			wantCSV: strings.Join([]string{
+				csv.FileHeader,
+				`Sensorservice Registration,string,1436292001000,1436292420000,"1,10182,civ,2500,0,Sensorservice Dump",`,
+				`Sensorservice Registration,string,1436291941000,1436292420000,"24,10013,com.google.android.gms.fitness.sensors.d.b,2500,0,Sensorservice Dump",`,
+				`Sensorservice Registration,string,1436290801000,1436292420000,"7,1000,com.android.server.display.AutomaticBrightnessController,2500,0,Sensorservice Dump",`,
+				`Sensorservice Registration,string,1436290211000,1436292420000,"4,10013,com.google.android.location.collectionlib.w,10000,0,Sensorservice Dump",`,
+				`Sensorservice Registration,string,1436290211000,1436292420000,"31,1000,com.android.server.policy.WindowOrientationListener,20000,0,Sensorservice Dump",`,
+			}, "\n"),
+			wantSensorErrors: []error{
+				fmt.Errorf("[Active Connection]: connection(5): the sensor(1) is not active according to the sensor device section"),
+			},
+		},
+		{
+			name: "[History] Test 3",
```

Comment:
Instead of numbering them (or maybe in addition to numbering), add a very short description of the test, or at the very least make a designation if the test data provides data with or without sensor errors.